elastic/kibana
# [Enterprise Search] Added Thumbnails to Search UI

## Summary

This PR adds thumbnail support to the Search UI page within App Search.

![latest](https://user-images.githubusercontent.com/1427475/124181721-e4da1300-da83-11eb-8fc3-392dc35a6ad3.gif)

Testing:
- Make sure that the thumbnails work, both when generating a UI and when downloading a ZIP file.
- Make sure that when the new Thumbnail dropdown is selected, that the correct portion of the image to the right of the form is highlighted

## Release note

Adds a Thumbnail Image option to the Search UI page within App Search.

### Checklist
